File: rb101/rb101-rb109-small_problems/easy2/retire.rb
print 'What is your age? '
current_age = gets.to_i

print 'At what age would you like to retire? '
retire_age = gets.to_i

current_year = Time.now.year
years_left = retire_age - current_age
retire_year = current_year + years_left

puts "It's #{current_year}. You will retire in #{retire_year}."
puts "You only have #{years_left} years of work to go!"

File: js210/js210-small_problems/easy3/retire.js
"use strict";

const rlSync = require('readline-sync');

const currentAge = parseInt(rlSync.question('What is your age? '), 10);
const retireAge = parseInt(rlSync.question('At what age would you like to retire? '), 10);
const remainingYears = retireAge - currentAge;

const currentYear = new Date().getFullYear();
const retireYear = currentYear + remainingYears;

console.log(`It's ${currentYear}. You will retire in ${retireYear}.`);
console.log(`You only have ${remainingYears} years of work to go!`);
